Resources/Private/Content/events.jsonl

```diff
@@ -1,211 +1,212 @@
-{"identifier":"2516b620-e6c9-4bda-b258-b1da34a8d937","type":"NodePeerVariantWasCreated","payload":{"contentStreamId":"1aae48c4-5028-430b-9ac4-41eff106dc4b","nodeAggregateId":"d17caff2-f50c-d30b-b735-9b9216de02e9","sourceOrigin":{"language":"de"},"peerOrigin":{"language":"en_UK"},"peerCoverage":[{"language":"en_UK"}]},"metadata":[]}
-{"identifier":"cb41fc13-bba6-4c42-b85c-0feb4beba15a","type":"NodePropertiesWereSet","payload":{"contentStreamId":"1aae48c4-5028-430b-9ac4-41eff106dc4b","nodeAggregateId":"d17caff2-f50c-d30b-b735-9b9216de02e9","originDimensionSpacePoint":{"language":"en_UK"},"affectedDimensionSpacePoints":[{"language":"en_UK"}],"propertyValues":{"title":{"value":"<h1>Create \u2764\ufe0f marvelous content.<\/h1>","type":"string"},"tagName":{"value":"h2","type":"string"}}},"metadata":[]}
-{"identifier":"923f899b-eff7-4195-8519-6019030f7ec9","type":"NodeGeneralizationVariantWasCreated","payload":{"contentStreamId":"1aae48c4-5028-430b-9ac4-41eff106dc4b","nodeAggregateId":"d17caff2-f50c-d30b-b735-9b9216de02e9","sourceOrigin":{"language":"en_UK"},"generalizationOrigin":{"language":"en_US"},"generalizationCoverage":[{"language":"en_US"}]},"metadata":[]}
-{"identifier":"5bf89af6-f3ad-434e-9516-77eba5bb6ff1","type":"NodePropertiesWereSet","payload":{"contentStreamId":"1aae48c4-5028-430b-9ac4-41eff106dc4b","nodeAggregateId":"d17caff2-f50c-d30b-b735-9b9216de02e9","originDimensionSpacePoint":{"language":"en_US"},"affectedDimensionSpacePoints":[{"language":"en_US"}],"propertyValues":{"title":{"value":"Welcome to the Neos CMS demo","type":"string"},"tagName":{"value":"h1","type":"string"}}},"metadata":[]}
-{"identifier":"d8ba380f-03a9-45ba-a1a1-8b65b4216c66","type":"NodeReferencesWereSet","payload":{"contentStreamId":"1aae48c4-5028-430b-9ac4-41eff106dc4b","sourceNodeAggregateId":"995c9174-ddd6-4d5c-cfc0-1ffc82184677","affectedSourceOriginDimensionSpacePoints":[{"language":"en_US"}],"referenceName":"metaNavigationItems","references":{"2d26b296-d3d2-69c1-b73e-0d1c801fe07a":{"targetNodeAggregateId":"2d26b296-d3d2-69c1-b73e-0d1c801fe07a","properties":null},"a56a3763-fa45-e12e-2236-5921e35ea756":{"targetNodeAggregateId":"a56a3763-fa45-e12e-2236-5921e35ea756","properties":null},"517ad799-35df-4324-9429-5c75629a8b34":{"targetNodeAggregateId":"517ad799-35df-4324-9429-5c75629a8b34","properties":null}}},"metadata":[]}
+{"identifier":"5561a3fc-09f6-4434-9604-57f611927a04","type":"ContentStreamWasCreated","payload":{"contentStreamId":"276e71e1-752b-46dd-b209-9c26f313450c"},"metadata":[]}
+{"identifier":"ddda7a2e-59ea-4202-bad6-3925e92e675b","type":"ContentStreamWasCreated","payload":{"contentStreamId":"276e71e1-752b-46dd-b209-9c26f313450c"},"metadata":[]}
```

Comment:
ah i see we duplicated the content stream again 😅 so we need https://github.com/neos/neos-development-collection/pull/4674  first before the export?